packages/connect-examples/expo-example/src/components/ui/AutoWrapperTextArea.tsx

```
@@ -1,21 +1,32 @@
-import { get } from 'lodash';
-import { useEffect, useRef } from 'react';
+import { isNaN } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)